discovery-provider/src/__init__.py

```diff
@@ -280,6 +280,8 @@ def configure_celery(flask_app, celery, test_config=None):
             if "url" in test_config["db"]:
                 database_url = test_config["db"]["url"]
 
+    ipld_interval = int(shared_config["discprov"]["blacklist_block_indexing_window"])
```

Comment:
is this a window or an interval?

Reply:
seems like it is named incorrectly - the "windows" refer to the # of blocks that are queued up and get processed, an interval is how frequently a task gets run. Let's fix this config name